specification/domainservices/resource-manager/Microsoft.AAD/stable/2021-03-01/oucontainer.json

```diff
@@ -0,0 +1,613 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Domain Services Resource Provider",
+    "description": "The AAD Domain Services API.",
+    "version": "2021-03-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Aad/operations": {
+      "get": {
+        "tags": [
+          "OuContainer"
+        ],
+        "description": "Lists all the available OuContainer operations.",
+        "operationId": "OuContainerOperations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          },
+          "200": {
+            "description": "HTTP 200 (OK) if the operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/OperationEntityListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Aad/domainServices/{domainServiceName}/ouContainer": {
+      "get": {
+        "tags": [
+          "OuContainer"
+        ],
+        "operationId": "OuContainer_List",

```

Comment:
What does Ou stands for? Per .NET convention, two letter acronym should be both capitalized.

Reply:
Organizational Unit.  This is something that is specific to HDI and AzureFiles, and no customer will ever have permissions to use.  And we're in the process of deprecating it, but we need our partners to make changes first.  I believe we already have exceptions for these issues, but I can follow-up with Jiho about it once he's back.